packages/checkout/order-shipping-packages/index.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * External dependencies
+ */
+import {
+	createSlotFill,
+	__experimentalUseSlot as useSlot,
```

Comment:
Since we can't get away without this hook, in a follow-up, I'm going to do a defensive check in case it's no longer `experimental` or if it's removed.